authz.go

```diff
@@ -65,6 +65,27 @@ func (a *authorizer) client(ctx context.Context) (*client, error) {
 	return newClient(a.Endpoint, a.Timeout, xp), nil
 }
 
+// clientX509 returns the client set up with x509 cert and key to make calls to Athenz.
+func (a *authorizer) clientX509(ctx context.Context) (*client, error) {
+
+	config, err := a.AthenzX509()
+	if err != nil {
+		return nil, err
+	}
+	xp509 := &http.Transport{
+		TLSClientConfig: config,
+	}
+	debugXp := &debugTransport{}
```

Comment:
Looks like we don't need this assignment here?

Reply:
@prabushyam I'm getting type mismatch otherwise.